engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthMask.java

```diff
@@ -61,10 +61,7 @@ public boolean isTheDefaultInstance() {
 
     @Override
     public boolean equals(Object obj) {
-        if (obj instanceof DisableDepthMask) {
-            return this.enabled == ((DisableDepthMask) obj).isEnabled();
-        }
-        return false;
+        return (obj instanceof DisableDepthMask) && this.enabled == ((DisableDepthMask) obj).isEnabled();
```

Comment:
Hmmm, I tried to avoid making these lines into one liner, do you think it's better in this way? _I suppose Intellij recommended this._